backend/RayTracing/Classes/Models/Object3D.py

```diff
@@ -8,7 +8,7 @@ def __init__(self, x=0, y=0, z=0, color=Color(), specular=100, reflection=0.1, t
         self.center = Vector(x, y, z)
         self.color = color
         self.specular = specular
-        self.reflection = reflection
+        self.reflection = self.reflectionCheck(reflection)
```

Comment:
This is an unnecessary change, reflectionCheck is not used anymore. There is no reflectionCheck implemented.

Reply:
Fixed :)